util/util.go

```diff
@@ -328,3 +330,56 @@ func PathParameters(endpoint, vaultPath string) (map[string]string, error) {
 	}
 	return result, nil
 }
+
+func ParseDurationSecond(in interface{}) (time.Duration, error) {
```

Comment:
This function was pulled from https://github.com/hashicorp/vault/blob/ed33ed1a0a5ecb9d41ab22b4899aa318239ac031/sdk/helper/parseutil/parseutil.go#L96-L147